docs/restate/cli.mdx

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_position: 4
+description: "Use the CLI to interact with Restate."
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# CLI 
+
+You can use the CLI to interact with Restate, and manage your services, deployments and invocations.
+
+## Installation
+
+Install the Restate CLI via:
+
+<Tabs groupId="operating-systems">
+<TabItem value="lin" label="npm">
+
+```shell
+npm install @restatedev/restate
+```
+
+</TabItem>
+<TabItem value="mac" label="Homebrew">
+
+```shell
+brew install restatedev/tap/restate
+```
+
+</TabItem>
+</Tabs>
+
+
+## Configuration
+
+There are two ways to configure the CLI: via environment variables or via a configuration file.
+
+### Using environment variables
+
+You can specify the following environment variables:
+
+- `RESTATE_HOST`: The hostname/IP address of the server. Default is `localhost`.
+- `RESTATE_HOST_SCHEME`: Default is `http`.
+- `RESTATE_META_URL`: To specify the full URL of meta service (scheme+host+port).
+- `RESTATE_DATAFUSION_HTTP_URL`: To specify the full URL of datafusion HTTP endpoint.
+- `RESTATE_AUTH_TOKEN`: Set if authentication is required.
+
+For example, to specify the hostname `myhost` and the host scheme `https`, pass environment variables as follows:
+
+```shell
+RESTATE_HOST=myhost RESTATE_HOST_SCHEME=https restate <command>
+```
+
+### Using a configuration file
+
+You can also specify the configuration in a `.env` file. The CLI will look for a `.env` file in its current directory.
+
+For example, to connect to a Restate meta running at `http://myhost:7777`, save the following in a `.env` file:
+
+```
+RESTATE_META_URL=http://myhost:7777
+```
+
+## Usage
+
+Here are a few useful commands:
```

Comment:
Suggested: `restate whoami` is a good way to know if your CLI is configured correctly or not.